includes/Story_Post_Type.php

```diff
@@ -398,6 +398,7 @@ public static function get_editor_settings() {
 					'fallbackPoster'  => plugins_url( 'assets/images/fallback-poster.jpg', WEBSTORIES_PLUGIN_FILE ),
 				],
 			],
+			'flags'  => [],
```

Comment:
Should this load from say a json file?

Reply:
That is mentioned as a possible next step in the PR description. So yes, maybe in the future we could do that.